dashboard/app/views/levels/_ai_tutor_panel.html.haml

```diff
@@ -1,5 +1,6 @@
 - data = {}
-- data[:level] = {type: @level&.type, hasValidation: !!@level&.properties["encrypted_validation"], id: @level&.id}
+- data[:level] = {id: @level&.id, type: @level&.type, hasValidation: !!@level&.properties["encrypted_validation"], isProjectBacked: !!@level&.properties["project_template_level_name"]}
+- data[:script_id] = @script.id
```

Comment:
TODO: we're going to need to handle `scriptId` and `isProjectBacked` in dashboard/app/views/levels/show.html.haml in https://github.com/code-dot-org/code-dot-org/pull/55937 if this PR goes in first.